src/main/java/seedu/guest/model/guest/Room.java

```diff
@@ -12,12 +12,16 @@ public class Room {
     public static final String MESSAGE_CONSTRAINTS =
             "Rooms should only contain alphanumeric characters and hyphens (-). "
             + "They cannot start or end with a hyphen. "
-            + "Hyphens can only be used between 2 alphanumeric characters.";
+            + "Hyphens can only be used between 2 alphanumeric characters."
+            + "You should enter a minimum of 1 character and a maximum of 100 characters.";
```

Comment:
Could we use passive voice instead? "They must be at least 1 character and at most 100 characters long." And insert this line before "They cannot start or end with a hyphen." as the length constraint is more important

Reply:
Agreed!